test/indexing.jl

```diff
@@ -253,4 +253,16 @@ using StaticArrays, Test
         @test lastindex(ind[3]) === 2
         @test size(ind[3]) === (2,)
     end
+
+    @testset "Array view into `Any` eltype `SArray`" begin
+        A = SVector{4, Any}(1,2,3,4)
+        v = @inferred view(A, SA[3, 1])
+        @test v == SVector{2, Any}(3, 1)
+        A = SMatrix{2, 2, Any}(1, 2, 3, 4)
+        v = @inferred view(A, @SArray(fill(1, 1)))
```

Comment:
```suggestion
        v = @inferred view(A, @SArray(fill(1, 1, 1)))
```
I think this needs one more `1`: two for dimensions and one for for actual element? (similarly in the next test)

Reply:
Looks like I confused myself - thanks for catching that.